_posts/2022-11-02-relicensing.md

```diff
@@ -0,0 +1,134 @@
+other permissive open source licenses. Ultimately, we do not want to
+tell our developers and users how to license their code but we also
+want to clearly signal that the work on MDAnalysis itself is
+guaranteed to remain open and free.
+
+
+## Why now the LGPL?
+
+We therefore decided to start the process to **relicense MDAnalysis
+under the Lesser GNU Public License v3 (LGPL)**. This open source
+license fulfills a number of important requirements for us:
+
+1. Downstream codes are able to freely import or link to MDAnalysis
+   library components without impacting the license choice of the
+   downstream code.
+   
+2. Downstream codes are able to use and subclass any MDAnalysis components
+   under its application programming interface (classes, methods, and
+   data objects), without impacting the license choice of the
+   downstream code.
+   
+3. Codes which either copy or extend the MDAnalysis library should
+   fall under the copyleft license requirements of the MDAnalysis
+   library license.
+   
+Thus, the LGPL gives people the freedom to choose any license for
+their own code that *makes use* of the MDAnalysis library as a whole
+(namely ``import MDAnalysis`` or subclassing). They can even make a
+commercial product out of it with a restrictive license (although we
+encourage open source licenses). However, one cannot just take parts
+of the MDAnalysis code and put it into another piece of code unless
+the other code is then *also* licensed under the GPL/LGPL. The LGPL
+ensures that the MDAnalysis library itself is guaranteed to remain
+open and free.
+
+We considered other licenses but none fulfilled our requirements.
+
+   
+## How will the relicensing process work?
+
+MDAnalysis has over 150 contributors. They all contributed code under
+the GPL v2+. In order to relicense, we need *all our
+contributors to agree to the license change*. 
+
+We know that this is a big change for the MDAnalysis community (and
+it's a big administrative and legal undertaking for us). We therefore
+want to give everyone an opportunity to ask questions so that you can
+make an informed decision.
+
+### Consultation period
+
+For the next month, the MDAnalysis Core Developers will answer your
+questions regarding the relicensing in our [public forums]({% link
+index.md %}#participating) (mailing lists, discord, twitter).
+
+### Contacting contributors
+
+After the consultation period, we will contact every contributor to
+MDAnalysis with a request to agree to changing the license from the
+current "GPL v2 or any later version" to "LGPL v3 or any later
+version".
+
+### License change
+
+Once we have the necessary agreement, we will switch over the license
+of MDAnalysis.
+
+### Timeline
+
+* November 2022: consultation period
+* December 2022: start contacting contributors
+* 2023?: We do not know how long the process will take. Our aim is to
+change the license as quickly as possible. We will keep the community
+updated.
```

Comment:
Todo - make sure we give a final date for the consultation period before we submit this

Reply:
yes